apps/docs/content/guides/database/custom-postgres-config.mdx

```diff
@@ -0,0 +1,150 @@
+| `track_io_timing`                    | Collects timing statistics for database I/O activity.                                                                                                        |
+
+For example, to enable `log_nested_statements` for the postgres role, execute:
+
+```sql
+alter role "postgres" set "auto_explain.log_nested_statements" to 'on';
+```
+
+To view the change:
+
+```sql
+select
+  rolname,
+  rolconfig
+from pg_roles
+where rolname = 'postgres';
+```
+
+### CLI configurable settings
+
+While many Postgres parameters are configurable directly, some configurations can be changed with the Supabase CLI at the [`system`](https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-SQL) level.
+
+<Admonition type="caution">
+
+CLI changes permanently overwrite default settings, so `reset all` and `set to default` commands won't revert to the original values.
+
+</Admonition>
+
+#### CLI supported parameters
+
+<Admonition type="tip">
+
+If a setting you need is not yet configurable, [share your use case with us](https://supabase.com/dashboard/support/new)! Let us know what setting you'd like to control, and we'll consider adding support in future updates.
+
+</Admonition>
+
+The following parameters are available for overrides:
+
+1. [effective_cache_size](https://postgresqlco.nf/doc/en/param/effective_cache_size/)
+1. [maintenance_work_mem](https://postgresqlco.nf/doc/en/param/maintenance_work_mem/)
+1. [max_connections](https://postgresqlco.nf/doc/en/param/max_connections/) (CLI only)
+1. [max_locks_per_transaction](https://postgresqlco.nf/doc/en/param/max_locks_per_transaction/) (CLI only)
+1. [max_parallel_maintenance_workers](https://postgresqlco.nf/doc/en/param/max_parallel_maintenance_workers/)
+1. [max_parallel_workers_per_gather](https://postgresqlco.nf/doc/en/param/max_parallel_workers_per_gather/)
+1. [max_parallel_workers](https://postgresqlco.nf/doc/en/param/max_parallel_workers/)
+1. [max_worker_processes](https://postgresqlco.nf/doc/en/param/max_worker_processes/) (CLI only)
+1. [session_replication_role](https://postgresqlco.nf/doc/en/param/session_replication_role/)
+1. [shared_buffers](https://postgresqlco.nf/doc/en/param/shared_buffers/) (CLI only)
+1. [work_mem](https://postgresqlco.nf/doc/en/param/work_mem/)
+
+#### Altering configs with the CLI
+
+To start:
+
+1. [Install](/docs/guides/resources/supabase-cli) Supabase CLI 1.69.0+.
+1. [Log in](/docs/guides/cli/local-development#log-in-to-the-supabase-cli) to your Supabase account using the CLI.
+
+To update Postgres configurations, use the [`postgres config`](https://supabase.com/docs/reference/cli/supabase-postgres-config) command:
+
+```bash
+$ supabase --experimental --project-ref <project-ref> postgres-config update --config shared_buffers=250MB
```

Comment:
```suggestion
supabase --experimental \
--project-ref <project-ref> \
postgres-config update --config shared_buffers=250MB
```

no need for the $ / # signs here. It will confuse many new devs